pkg/services/thumbs/crawler.go

```diff
@@ -29,19 +34,21 @@ type simpleCrawler struct {
 	threadCount       int
 
 	glive  *live.GrafanaLive
+	store  *sqlstore.SQLStore
 	mode   CrawlerMode
 	opts   rendering.Opts
 	status crawlStatus
 	queue  []dashItem
 	mu     sync.Mutex
 }
 
-func newSimpleCrawler(folder string, renderService rendering.Service, gl *live.GrafanaLive) dashRenderer {
+func newSimpleCrawler(folder string, renderService rendering.Service, gl *live.GrafanaLive, store *sqlstore.SQLStore) dashRenderer {
```

Comment:
should I depend on `sqlstore.SQLStore` and use it directly or should I send commands to it through the `bus`? if the latter, does the same rule apply to `rendering.Service` and `live.GrafanaLive`? @ryantxu @FZambia

Reply:
we should avoid using `bus` where possible -- now that we can inject explicit interfaces (wire) we are slowly removing bus usage so the real dependencies are more discoverable.